ui/app/utils/default_jobs/actions.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+export default `job "redis-actions" {
+  datacenters = ["*"]
+
+  group "cache" {
+    count = 1
+    network {
+      port "db" {}
+    }
+
+    task "redis" {
+      driver = "docker"
+
+      config {
+        image = "redis:3.2"
+        ports = ["db"]
+        command = "/bin/sh"
+        args = ["-c", "redis-server --port \${NOMAD_PORT_db} & /local/db_log.sh"]
+      }
+
+      resources {
+        cpu = 500
+        memory = 256
+      }
+
+      service {
+        name = "redis-service"
+        port = "db"
+        provider = "nomad"
```

Comment:
```suggestion
      config {
        image   = "redis:3.2"
        ports   = ["db"]
        command = "/bin/sh"
        args    = ["-c", "redis-server --port \${NOMAD_PORT_db} & /local/db_log.sh"]
      }

      resources {
        cpu    = 500
        memory = 256
      }

      service {
        name     = "redis-service"
        port     = "db"
        provider = "nomad"
```

![image](https://github.com/hashicorp/nomad/assets/775380/7087d6c2-345e-492c-9cc3-6660c48ae083)